lazy-pages/src/lib.rs

```diff
@@ -110,7 +111,7 @@ pub fn initialize_for_program(
     wasm_mem_addr: Option<usize>,
     wasm_mem_size: u32,
     stack_end: Option<u32>,
-    program_id: Vec<u8>,
+    program_id: ProgramId,
```

Comment:
I would ask to keep the type as `Vec<u8>` since there are plans to keep programs memory in triple map storage. Frankly speaking it would be more preferred to name this arg as `program_prefix`.

Reply:
Agree. How about letting it be `program_key`?